chicago/bills.py

```diff
@@ -160,40 +86,144 @@ def scrape(self):
+                              vote['Person Name']['label']))
+
+        action_detail_div = action_detail_page.xpath(".//div[@id='ctl00_ContentPlaceHolder1_pageTop1']")[0]
+        action_details = self.parseDetails(action_detail_div)
+        result = action_details['Result'].lower()
+
+        return result, vote_list
+
+
+
+    def addBillHistory(self, bill, history_table) :
+        all_votes = []
+        
+        history = self.parseDataTable(history_table)
+
+        for action, _, _ in history :
+            action_description = action['Action']
+            action_date =  action['Date'].date().isoformat()
+            try :
+                if action_description :
+                    bill.add_action(action_description,
+                                    action_date,
+                                    organization=action['Action\xa0By'],
+                                    classification=ACTION_CLASSIFICATION[action_description])
+                    if 'url' in action['Action\xa0Details'] :
+                        action_detail_url = action['Action\xa0Details']['url']
+                        result, votes = self.extractVotes(action_detail_url)
+
+                        if votes :
+                            action_vote = Vote(legislative_session=self.session, 
+                                               motion_text=action_description,
+                                               classification='bill-passage',
+                                               start_date=action_date,
+                                               result=result,
+                                               bill=bill.identifier)
+                            action_vote.add_source(action_detail_url)
+                            for option, voter in votes :
+                                action_vote.vote(option, voter)
+                        
+                            all_votes.append(action_vote)
+
+            except KeyError :
+                if action_description not in ('Direct Introduction',
+                                              'Remove Co-Sponsor(s)',
+                                              'Held in Committee',
+                                              'Deferred and Published') :
+                    print(action_description)
+                    raise
+
+
+        return all_votes
+
+
+    def addDetails(self, bill, detail_url) :
+        detail_page = self.lxmlize(detail_url)
+        detail_div = detail_page.xpath(".//div[@id='ctl00_ContentPlaceHolder1_pageDetails']")[0]
+
+        legislation_details = self.parseDetails(detail_div)
+        
+
+        for related_bill in legislation_details.get('Related files', []) :
+            # need different relation_type
+            bill.add_related_bill(identifier = related_bill['label'],
+                                  legislative_session = self.session,
+                                  relation_type='replaces')
```

Comment:
We need a relation type that will let us handle https://github.com/opencivicdata/pupa/issues/62 and https://github.com/opencivicdata/pupa/issues/86